Core/Object Arts/Dolphin/Base/ClassBuilder.cls

```diff
@@ -488,10 +488,15 @@ mutateInSitu
 	"Private - Mutate the class being modified without mutating its instances.
 	In fact, we only need to change the superclass."
```

Comment:
This comment was a correct description of the actual behaviour, but not the desired behavior. With your change it no longer describes either :-)